danielMunteanu/src/main/java/danielCollectionTask/FilterByType.java

```diff
@@ -0,0 +1,20 @@
+package danielCollectionTask;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class FilterByType {
+    public static List<UserOrder> filterOrderByItemType(List<UserOrder> userOrderList) {
```

Comment:
We have 2 different order types, could you please apply some SOLID and OOP principles in order to make it fine for CompanyOrder too?

Reply:
fixed